src/ILLink.RoslynAnalyzer/Constants.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace ILLink.RoslynAnalyzer
+{
+	public static class Constants
+	{
+		public static class WarningCodes
+		{
+			public const string IL2026 = nameof (IL2026);
```

Comment:
For me seems like this list of warnings will be replaced in the future with a shared enum with the linker, but for now seems ok.

Reply:
Yup. That's the plan.